pylabrobot/resources/thermo_fisher/README.md

```diff
@@ -1,18 +1,37 @@
-# Resource definitions: Thermo Fisher
+# Resource definitions: Thermo Fisher Scientific Inc.
 
-Company page: [Thermo Fisher Scientific Wikipedia](https://en.wikipedia.org/wiki/Thermo_Fisher_Scientific)
+Company page: [Thermo Fisher Scientific Inc. Wikipedia](https://en.wikipedia.org/wiki/Thermo_Fisher_Scientific)
 
 > Thermo Fisher Scientific Inc. is an American supplier of analytical instruments, life sciences solutions, specialty diagnostics, laboratory, pharmaceutical and biotechnology services. Based in Waltham, Massachusetts, Thermo Fisher was formed through the **merger of Thermo Electron and Fisher Scientific in 2006**. Thermo Fisher Scientific has acquired other reagent, consumable, instrumentation, and service providers, including Life Technologies Corporation (2013), Alfa Aesar (2015), Affymetrix (2016), FEI Company (2016), BD Advanced Bioprocessing (2018),and PPD (2021).
 
 > As of 2023, the company had a market capitalization of $202 billion. It ranked 97th on the Fortune 500 list based on its 2022 annual revenue of US$44.92 billion.
 
+A basic structure of the companiy, [its brands](https://www.thermofisher.com/uk/en/home/brands.html) and product lines looks like this:
+
+```rust
+Thermo Fisher Scientific Inc. (TFS, aka "Thermo")
+├── Applied Biosystems (AB; brand)
+│   └── MicroAmp
+│      └── EnduraPlate
+├── Fisher Scientific (FS; brand)
+├── Invitrogen (INV; brand)
+├── Ion Torrent (IT; brand)
+├── Gibco (GIB; brand)
+├── Thermo Scientific (TS; brand)
+│   ├── Nalgene
+│   ├── Nunc
+│   └── Pierce
+├── Unity Lab Services (brand, services)
+├── Patheon (brand, services)
+└── PPD (brand, services)
+```
```

Comment:
I think this might the first clear public explanation of what is going on here :)

Reply:
Hahaha thanks...
To be honest, all these mergers and acquisitions and what TFS then actually does with the companies it sucked up is very hard to see through, but I'm quite confident in this structure now.

I will similarly reorganise Corning resources... Similar behemoth to TFS

Reply:
I left out Life Technologies because it's not a brand anymore and is usually just found in old docs as a relic (please correct me if you find evidence to the contrary)